apps/main/utils.py

```diff
@@ -51,3 +53,17 @@ def width_to_size(size_list):
             item = item.replace(';', '')
             new_size_list.append(item)
     return new_size_list
+
+
+def upload_files(item, file):
```

Comment:
If you encounter cycling import issue, let's think about some modules restructure.